### PR TITLE
Fix kafka input binding looping when not topics are set

### PR DESCRIPTION
**Pull request by Gehhilfe**
Signed-off-by: Tim Burkert <burkert.tim@gmail.com>

# Description

Prevent kafka input binding reading when no topics are provided. When not or only malformed topics are provided, the read binding will loop forever and waste resources. 

image

## Issue reference

Please reference the issue this PR will close: #1399 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by Taction** [comment]
Thanks for your contribution! LGTM overall, just a small change.

**Comment by yaron2** [comment]
> Thanks for your contribution! LGTM overall, just a small change.

Did you submit your review? I don't see any comments.

**Review by Taction — CHANGES_REQUESTED** [review]

**Comment by Taction** [comment]
> > Thanks for your contribution! LGTM overall, just a small change.
> 
> Did you submit your review? I don't see any comments.

Thanks, I have forgotten to submit it.

File: bindings/kafka/kafka.go [hunk 1]
```diff
@@ -226,6 +226,10 @@ func (k *Kafka) getSyncProducer(meta *kafkaMetadata) (sarama.SyncProducer, error
 }
 
 func (k *Kafka) Read(handler func(*bindings.ReadResponse) ([]byte, error)) error {
+	if len(k.topics) == 0 {
+		return errors.New("kafka error: no topics configured")
```
[the remark this answers is not in the document]
Comment (Gehhilfe) [reply bindings/kafka/kafka.go]: Done

**Review by artursouza — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]

**Comment by yaron2** [comment]
Thanks @Gehhilfe for this PR!

**Pull request merged**